### PR TITLE
Support css modules

### PR DESCRIPTION
**Pull request by loklaan**
Hey mate,

Your package for autocompleting modules is simple fit my use case for [css-modules dependencies](https://github.com/css-modules/css-modules#dependencies) autocompletion.

But, this may go beyond the advertised scope for your package so no pressure to accept PR.

### CODE REVIEW

**Comment by nkt** [comment]
Hm, Good idea, but I think we should check, if we are inside js file, check `require` and `import`, and if in `css`, check `composes`

**Comment by loklaan** [comment]
You're right. Do you want me to generate a new gif for the readme? I'll attach it in comments here before putting it in the repo to avoid binary bloat.

File: src/completion-provider.js [hunk 1]
```diff
@@ -9,15 +9,27 @@ const internalModules = require('./internal-modules');
...
   }
 
-  getSuggestions({editor, bufferPosition, prefix}) {
+  getSuggestions({editor, bufferPosition, prefix, scopeDescriptor}) {
     const line = editor.getTextInRange([[bufferPosition.row, 0], bufferPosition]);
-    if (!/require|import/.test(line)) {
-      return [];
+
+    const sourceLanguage = scopeDescriptor.scopes[0]
+    switch (sourceLanguage) {
```
Comment (nkt) [inline src/completion-provider.js]: Please do not copy-paste.
Add function `getTestRegExp(sourceLanguage)`, for example.

Comment (loklaan) [inline src/completion-provider.js]: Fair enough! I've cleaned that logic up.

**Comment by nkt** [comment]
Great, thanks!

**Comment by nkt** [comment]
@loklaan your changes have broken this plugin, so I've reverted this PR.

**Comment by loklaan** [comment]
No worries. I'll eventually get around to fixing this, or making another package for richer css module support.

**Pull request merged**